src/test/java/de/tum/in/www1/artemis/localvcci/LocalVCIntegrationTest.java

```diff
@@ -223,7 +223,7 @@ void testUserCreatesNewBranch() throws Exception {
         Path testFilePath = assignmentRepository.localRepoFile.toPath().resolve("new-file.txt");
         Files.createFile(testFilePath);
         assignmentRepository.localGit.add().addFilepattern(".").call();
-        assignmentRepository.localGit.commit().setMessage("Add new file").call();
+        assignmentRepository.localGit.commit().setSign(false).setMessage("Add new file").call();
```

Comment:
The newly extracted method isn't called in here.

Reply:
thanks, I changed it